### ISSUE TITLE
Toegankelijkheidsverbeteringen Q2-2025

### ISSUE THREAD

**Issue by joeribekker** [open-formulieren/open-forms#5138]
November 26th a new Accessibility test was performed which resulted into a new rapport. Most items in this rapport are 'small' improvements (color contrasts, changing html elements), but there are also a couple 'larger' issues. The list below contains the rapported accessibilty items, which I've tests with the macOS VoiceOver screenreader:

**Improvements that should be relatively simple:**
- [ ] 1.1.1 (non-textual content) The logo in the pdf contains a textual alternative, but cannot be accessed.
- [ ] 1.3.1 (information and relations) Missing relations between (submission rapport) PDF field labels and values.
- [ ] 1.3.1 (information and relations) When a (submission rapport) PDF field doesn't have a values, we should provide a "No information provided" alternative. (This is to ensure that users understand that a field was left empty)
- [ ] 1.4.3 (color contrast) Error messages have a contrast of 3,7:1. This should be 4,5:1
- [ ] 1.4.11 (non-textual color contrast) Checkbox and radio button borders have a contrast of 2,9:1. This should be 3,0:1
- [ ] 2.1.1 (navigation - keyboard) Tooltip cannot be accessed by screen readers
- [ ] 2.4.2 (navigation - page title) The title in the PDF meta description contains leading and trailing whitespace
- [ ] 3.1.2 (language) Some texts are non translatable. The rapport mentions the `title` of the select component delete button.
- [ ] 4.1.2 (name, role, value) The dropdown variant of the progress menu doesn't contain `aria-expanded` information

All these smaller improvements should be completable in around 3 days

**Improvements that are a bit more complicated:**
- [ ] 2.4.7 (navigation - focus visibility) When the form is displayed on a small screen, or greatly zoomed-in, the progress menu will be shown as a dropdown menu. When you focus any menu items in the dropdown, and repeatable press the tab key, you can place the focus outside the menu. The menu stays expended, resulting in the focused element not being visible.
This might required some major re-writing.
Could take a couple of days
- [ ] 3.3.3 (error suggestion) When a specific input pattern is expected, the error message should provide the expected format or an example.
With examples in the error messages this could be resolved in 1 day.
- [ ] 4.1.2 (name, role, value) The select component contains problems with `name` and `role` on the select element, `aria-selected="true"` on the parent div, missing aria tags
This could be resolved in 2 days (It could be that we cannot fix the issues for this component. Lets start with a time-box of 1/2 day, and assess from there)
- [ ] 4.1.3 (status messages) The loading spinners have issues with `aria-live` which results into status updates not being announced
This could be resolved in about 1 week

Estimate (including previously done work): 6 weeks
